src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -152,35 +112,76 @@
  * Example:
  * </p>
  * <pre>
- * switch (i) {
- *   case 1:
- *     break;
- *   case 2: // Previous case: OK
- *           // This case: violation, last case must have statement that transfer control
- *     i++;
+ * class Example2 {
+ *   public int foo() {
+ *     int i = 0;
+ *     while(i &gt;= 0) {
+ *       switch(i) {
+ *         case 1:
+ *           i++; // violation
+ *         case 2:
+ *           i++;
+ *           break;
+ *         case 3:
+ *           i++;
+ *           return;
+ *         case 4:
+ *           i++;
+ *           throw new Exception();
+ *         case 5:
+ *           i++;
+ *           continue;
+ *         case 6:
+ *         case 7:
+ *         case 8:
+ *           i++; // no break by design
```

Comment:
@romani. Do you want this comment to be replaced by `// violation` since Example 2 is depicting lastCaseGroup scenario?

Reply:
Should stay. All except for "violation" comment should absolutely the same.